packages/server/src/validator/utils.ts

```diff
@@ -27,7 +35,7 @@ export function validateParam(index: number | string, param: any, validation: an
     }
   }
 
-  if (param != null && !Array.isArray(paramType)) {
+  if (!Array.isArray(paramType)) {
     const result = isArray ? paramType.test(index, param, validation.type[1]) : paramType.test(param);
```

Comment:
We already know `isArray === false` here
```suggestion
    const result = : paramType.test(param);
```

Reply:
Fixed it